File: 1223/array1223/src/array1223/Arr1.java
package array1223;

public class Arr1 {

	public static void main(String[] args) {
		int arr[] = new int[100];
		for (int i = 0; i < arr.length; i++) {
			arr[i] = (i + 1);
			if ((i + 1) % 5 == 0) {
				System.out.println(arr[i]);
			}
			
			if (arr[i] > 89) {
				System.exit(0);
			}

		}

	}
}

File: 0208/class0208/Array/Array/Arr1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Array
{
    class Arr1
    {
        private int hap;
        private int avg;
        int[] arr = { 10, 20, 30, 40, 50, 60, 70, 80, 90, 100 };

        public Arr1(int hap, int avg)
        {
            this.hap = hap;
            this.avg = avg;
        }

        public int Hap
        {
            get { return hap; }
            set { hap = value; }
        }

        public int Avg
        {
            get { return avg; }
            set { avg = value; }
        }
    }
}
